pkg/natsreloader/natsreloader.go

```diff
@@ -374,3 +393,93 @@ func retryJitter(base time.Duration) time.Duration {
 	offset := rand.Float64()*0.2 - 0.1
 	return time.Duration(b + offset)
 }
+
+func getServerFiles(configFile string) ([]string, error) {
+	filePaths, err := getIncludePaths(configFile, make(map[string]interface{}))
+	if err != nil {
+		return nil, err
+	}
+
+	certPaths, err := getCertPaths(filePaths)
+	if err != nil {
+		return nil, err
+	}
+
+	filePaths = append(filePaths, certPaths...)
+	sort.Strings(filePaths)
+
+	return filePaths, nil
+}
+
+func getIncludePaths(configFile string, checked map[string]interface{}) ([]string, error) {
+	if _, ok := checked[configFile]; ok {
+		return []string{}, nil
+	}
+
+	configFile, err := filepath.Abs(configFile)
+	if err != nil {
+		return nil, err
+	}
+
+	// Add the current config file to the list and mark it as checked
+	filePaths := []string{configFile}
+	checked[configFile] = nil
+
+	parentDirectory := filepath.Dir(configFile)
+	includeRegex := regexp.MustCompile(`(?m)^\s*include\s+([^\n]*)`)
```

Comment:
I think we may have to handle quoted values here? to support:
```
include "foo.conf"
include foo.conf
include 'foo.conf'
```
maybe something like this?
```
(?m)^\s*include\s+'?"?([^"?'?\n]*)
```

Reply:
This regex will actually already accept quoted strings, once it matches `include ` it will greedily match any characters not in `[^\n]` so it will match everything until it hits a newline character

Reply:
Oh, actually you are correct because it will include the quotes as part of the filename - I will update